install.sh

```diff
@@ -87,33 +190,22 @@ can be used to login directly using psql or another client.
 --------------------------------------------------------------------------------
 
 "
-read -r -p "Choose a database name [mathesar]: " db_name
-db_name=${db_name:-mathesar}
 
-read -r -p "Choose a username [mathesar]: " db_username
-db_username=${db_username:-mathesar}
+existing=$(standardize_yesno "Do you have a preexisting PostgreSQL database to connect?")
```

Comment:
I would say something like "Would you like to connect an existing database or create a new database?" here. It's more clear.

Reply:
I'm not sure how to interpret the result. I wanted a "yes-no" question

Reply:
Okay, I changed this to use a select menu.